src/services/__tests__/public-holidays.service.test.ts

```diff
@@ -0,0 +1,125 @@
+import axios from "axios";
+import {
+  getListOfPublicHolidays,
+  checkIfTodayIsPublicHoliday,
+  getNextPublicHolidays,
+} from "../public-holidays.service";
+import holidaysMock from "../mockResponse.json";
+import holidaysFormatted from "../mockFormattedResponse.json";
+import { shortenPublicHoliday, validateInput } from "../../helpers";
+import { PUBLIC_HOLIDAYS_API_URL } from "../../config";
+
+jest.mock("../../helpers", () => ({
+  validateInput: jest.fn(() => true),
+  shortenPublicHoliday: jest.fn((data) => data),
+}));
+
+describe("Servises tests", () => {
+  describe("getListOfPublicHolidays tests", () => {
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it("should trigger call with expected params", async () => {
+      const year = 2023;
+      const country = "GB";
+      const axiosGetSpy = jest
+        .spyOn(axios, "get")
+        .mockImplementation(() => Promise.resolve({ data: holidaysMock }));
```

Comment:
this is a wrapper for `mockImplementation` that automatically resolves
```suggestion
        .mockResolvedValue({ data: holidaysMock });
```

Reply:
fixed